src/unstract/sdk/adapters/x2text/llm_whisperer_v2/src/helper.py

```diff
@@ -341,7 +347,7 @@ def write_output_to_file(output_json: dict, output_file_path: Path) -> None:
         try:
             text_output = output_json.get("result_text", "")
             logger.info(f"Writing output to {output_file_path}")
-            output_file_path.write_text(text_output, encoding="utf-8")
+            fs.write(data=text_output, encoding="utf-8")
```

Comment:
@harini-venkataraman I think the path is missing in these calls

Reply:
Addressed